### ISSUE TITLE
Upgrade to Pyodide 0.21.0

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-lite#41]
> Just spotted this [in the docs](https://pyodide.org/en/stable/usage/loading-packages.html#installing-wheels-from-arbitrary-urls):
>
>> Micropip will also install the dependencies of the wheel. If dependency resolution is not desired, you may pass `deps=False`.
>
> Looks like that's new in Pyodide 0.21.0 (I am on 0.20.0 right now): https://pyodide.org/en/stable/project/changelog.html#version-0-21-0

_Originally posted by @simonw in https://github.com/simonw/datasette-lite/issues/40#issuecomment-1215943203_

**Comment by simonw** [comment]
File sizes are similar (comparing the two):

<img width="986" alt="image" src="https://user-images.githubusercontent.com/9599/184736372-2f4dff08-28ef-457c-8c5b-5f0e86893cd8.png">

**Comment by simonw** [comment]
I reverted this because it may have caused this issue:
- #42

**Comment by rclement** [comment]
PR #64 contains the upgrade to Pyodide 0.22.1 (latest 0.22.x version), working well on my end.
Notable changes: 
- Requires `loadPyodide({fullStdLib: true})`, introduced with 0.21.1
- Includes `sqlite` 3.39.0 with the bundled JSON extension

**Comment by rclement** [comment]
Pyodide > 0.23.0 is currently not compatible with `datasette-lite`, throwing the following error:

```python
Traceback (most recent call last):
  File "/lib/python311.zip/_pyodide/_base.py", line 540, in eval_code_async
    await CodeRunner(
  File "/lib/python311.zip/_pyodide/_base.py", line 365, in run_async
    await coroutine
  File "<exec>", line 103, in <module>
  File "/lib/python3.11/site-packages/datasette/__init__.py", line 2, in <module>
    from datasette.utils.asgi import Forbidden, NotFound, Request, Response  # noqa
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lib/python3.11/site-packages/datasette/utils/asgi.py", line 8, in <module>
    import aiofiles.os
  File "/lib/python3.11/site-packages/aiofiles/os.py", line 31, in <module>
    link = wrap(os.link)
                ^^^^^^^
AttributeError: module 'os' has no attribute 'link'
```

**Comment by simonw** [comment]
Closed in favour of #64
